app.py

```diff
@@ -68,6 +69,15 @@ def create_plant(user_id):
         'care': request.json.get('care', {})
     }
     mongo.db.plants.insert_one(plant)
+
+    journal_entry = {
+        '_id': str(ObjectId()),
+        'plant_id': plant['_id'],
+        'entry_type': 'image',
+        'timestamp': timestamp,
+        'info': plant['image_url'],
```

Comment:
This needs to be nested inside an object to match with our schema
```
'info': {
    'imageURL': plant['image_url'],
    'notes': 'Plant added to garden!'
}
```

Reply:
I updated it to account for the nested URL :D